packages/format-library/src/link/style.scss

```diff
@@ -11,4 +11,8 @@
 	white-space: nowrap;
 	min-width: 150px;
 	max-width: 500px;
+
+	&.has-invalid-link {
+		color: $alert-red;
```

Comment:
This should be limited only to the link which is incorrect:

![screen shot 2018-11-09 at 12 08 54](https://user-images.githubusercontent.com/699132/48259454-443d0d80-e418-11e8-8474-be057eec2eaa.png)

Reply:
Oops, it's my browser highlighting spelling errors :P